Extractor/Subscriptions/BaseCreateSubscriptionTask.cs

```diff
@@ -130,13 +130,26 @@ private async Task<Subscription> EnsureSubscriptionExists(ILogger logger, Sessio
 
             if (subscription == null)
             {
+                uint maxNotificationsPerPublish = 0;
+                if (SubscriptionName.IsEvents())
+                {
+                    maxNotificationsPerPublish = config.Subscriptions.MaxEventNotificationsPerPublish;
+                }
+                else if (SubscriptionName.IsDataPoints())
+                {
+                    maxNotificationsPerPublish = config.Subscriptions.MaxDataNotificationsPerPublish;
+                }
+                // For general subscriptions, the default value of 0 is going to be fine. In practice,
+                // we will never receive that many notifications for those, since they are very limited in size.
```

Comment:
This comment confused me a bit
1- Sounds like we will never recieve as many as 0 notifications
2- Can we clarify size of what we are referring to
```suggestion
                // For general subscriptions, the default value of 0 is going to be fine. In practice,
                // we will never receive many notifications for datapoints or events, since they are very limited in size.
```

Reply:
Rewrote the comment